docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,57 @@
+# Experimental Features
+
+Cluster API now ships with a new experimental package that lives under exp/ directory which has new features. This is a
+temporary location for features which will be moved to their permanent locations after graduation. Users can experiment with these features by enabling them using feature gates.
+
+## Enabling Experimental Features for Management Clusters Started with clusterctl
+
+Users can enable/disable features using OS environment variables (`export EXP_CLUSTER_RESOURSE_SET=true`), but it is also possible to set them in the clusterctl config file located at`$HOME/.cluster-api/clusterctl.yaml`:
+
+```yaml
+# Values for environment variable substitution
+EXP_CLUSTER_RESOURCE_SET: "true"
+```
+In case a variable is defined both in the config file and as an OS environment variable, the latter takes precedence.
+For more information on how to set variables for clusterctl, see [clusterctl Configuration File](../../clusterctl/configuration.md)
+
+Some features like `MachinePools` may require infrastructure providers to implement a separate CRD that handles infrastructure side of the feature too. 
+For such a feature to work, both cluster api and infrastructure provider should enable the corresponding features. Infrastructure providers may use different variables to enable such features.
```

Comment:
If I'm not wrong this should not be the case anymore because we agreed on the same variable name

Reply:
Here I am trying to explain infra providers also may implement their side of features as features. Rephrased.